code/COVID19 US county Map-Daily Update.ipynb

```diff
@@ -361,16 +385,16 @@
        "1      13141    40665  "
       ]
      },
-     "execution_count": 158,
+     "execution_count": 170,
      "metadata": {},
      "output_type": "execute_result"
     }
    ],
    "source": [
     "#census and health data from ESRI\n",
-    "df_ACS1=pd.read_excel(covid19Root + r'/ACS_2014-2018_Fields.xlsx')\n",
-    "df_ACSState=pd.read_excel(covid19Root + r'/ACS_State_Final_ToExcel_noMOE.xlsx')\n",
-    "df_ACSCounty=pd.read_excel(covid19Root + r'/ACS_County_Final_ToExcel_noMOE.xlsx')\n",
+    "df_ACS1=pd.read_excel(covid19Root / 'ACS_2014-2018_Fields.xlsx')\n",
```

Comment:
If you do what I suggest above this would become covid19_dataTables (or somesuch)